DAO/MessageDAO.py

```diff
@@ -161,3 +161,19 @@ def insert_reply(self, message, uid, mid, cid, img=None):
         self.conn.commit()
         return reply_id
 
+    def remove_vote(self, mid, uid, upvote):
+        cursor = self.get_cursor()
+        query = "SELECT * FROM vote WHERE mid = %s and uid = %s"
+        cursor.execute(query, (mid, uid))
+        vote = [vote for vote in cursor.fetchall()]
```

Comment:
I'm assuming that there can only be one vote per user. In that case you can do the following

```python
delete = cursor.fetchone()['upvote'] == upvote
```

Reply:
Will the way I did it affect functionality?

Reply:
Noted